v2.6/getting-started/kubernetes/upgrade/convert.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Converting policies
+redirect_from: latest/getting-started/kubernetes/upgrade/convert
+---
+
+If you are upgrading from the following, you must convert your policies
+before you can upgrade the Calico components.
+
+| Current version     | Datastore type           | Conversion required? |
+| ------------------- | ------------------------ | -------------------- |
+| earlier than v2.3.0 | Kubernetes API datastore | Yes                  |
+| earlier than v2.4.0 | etcd                     | Yes                  |
+
+Calico's interpretations of policy [changed]((https://github.com/kubernetes/kubernetes/pull/39164#issue-197243974)) 
```

Comment:
This should indicate that it occurred to match a change to the upstream APIs behavior.

Reply:
This should be more specific - "Calico's interpretation of Kubernetes NetworkPolicies changed to match the behavior defined upstream"

More generally, Calico's implementation of policy hasn't changed. Just the semantics on a particular API.